symfit/core/fit.py

```diff
@@ -1312,31 +1312,34 @@ def __init__(self, model, *ordered_data, **named_data):
                 minimizer = BFGS
 
         # Initialise the minimizer
-        if isinstance(minimizer, BaseMinimizer):
-            self.minimizer = minimizer
+        if isinstance(minimizer, Sequence):
```

Comment:
Nice job on isolating the setup. Not sure I understand the comment though, why should we delay the setup of jacobians to ChainedMinimizer? Looks very clean the way it is right?

Reply:
Fully agree on second thought